lib/npo.src.js

```diff
@@ -12,12 +12,59 @@
 	/*jshint validthis:true */
 	"use strict";
 
-	var builtInProp, cycle, scheduling_queue,
+	var builtInProp, cycle, scheduling_queue, nextShouldNotEmit = false,
```

Comment:
code style in this file has variable names as "snake case", not camel case. let's be consistent.